### DISCUSSION TITLE
Docs Multi-instance breadcrumb improvement

### DISCUSSION THREAD

**Discussion by plamenkoyovchev** [facebook/docusaurus#9898] [Ideas]
Hi!

I'm using docs multi-instance and I've noticed something which I didn't expect. You can also check it on your website.

[Docs](https://docusaurus.io/docs)

and 

[Community](https://docusaurus.io/community)

Since Docs and Community are two different instances shouldn't the Breadcrumb home link for Docs instance:

![image](https://github.com/facebook/docusaurus/assets/3595159/70ee8c42-78ac-4ff6-9795-c61a14546803)

be: `https://docusaurus.io/docs` instead of `https://docusaurus.io/` ? 

also 

the Breadcrumb home link for Community instance:

![image](https://github.com/facebook/docusaurus/assets/3595159/70ee8c42-78ac-4ff6-9795-c61a14546803)

be: `https://docusaurus.io/community` instead of `https://docusaurus.io/` ?

**Comment by slorber** [comment]
You are right, these improvements are being discussed in https://github.com/facebook/docusaurus/issues/6953
